docker-compose.production.yml

```
@@ -0,0 +1,75 @@
+    build:
+      context: .
+      dockerfile: Dockerfile.production
+    image: ch11-backfill-worker:latest
+    command: ["python", "-m", "src.workers.backfill_worker"]
+    environment:
+      - DATABASE_URL=${DATABASE_URL}
+      - S3_BUCKET=ch11-bankruptcy-documents
+      - SQS_QUEUE_URL=${SQS_BACKFILL_QUEUE_URL}
+      - LOG_LEVEL=INFO
+    deploy:
+      replicas: 10  # Scale up for backfill processing
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)